src/System.Device.Gpio/System/Device/Pwm/Channels/Windows10PwmChannel.Windows.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
```

Comment:
Can we rename Windows10PwmChannel to GenericPwmChannel or DefaultPwmChannel and use one code on one platform and the other on other? (this way we have 2 files less)

Reply:
Given they are internal we don't have to have the extra files so I don't mind having one called Windows and the other Unix.